openmetadata-ui/src/main/resources/ui/src/components/Team/TeamDetails/UserTab/UserTab.component.tsx

```diff
@@ -274,19 +297,20 @@ export const UserTab = ({
         )}
       </Col>
 
-      <ConfirmationModal
-        bodyText={t('message.are-you-sure-want-to-text', {
+      <Modal
```

Comment:
Why did we replace confirmation modal here?

Reply:
I saw the ConfirmationModal component has been deprecated that's why I replaced it